game_eggs/steamcmd/valheim/vanilla/README.md

```diff
@@ -0,0 +1,26 @@
+## Server Ports
+
+Ports are required to run the server in a table format.
+
+| Ports             | Default |
+| ----------------- | ------- |
+| Game              | 2456   |
+| Reserved (Game+1) | 2457   |
+
+# Authors / Contributors
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+    <tr>
+        <td align="center">
+            <a href="https://github.com/NHDriver4">
```

Comment:
Thats the Egg from Parkers Repo and its released under MIT, so you have to provide the Licence File with the original Author or you are not really allowed to change the Author

Reply:
Got it. It was just a copy paste tbh. I'll resolve this later.

Reply:
The license file has been posted. That should cover this right? Sorry, I'm a bit new to all of this and still learning git etiquette!